cmd/release/cmd/list.go

```diff
@@ -40,8 +41,38 @@ var rancherListRCDepsSubCmd = &cobra.Command{
 	},
 }
 
+var chartsListSubCmd = &cobra.Command{
+	Use:   "charts [branch] [charts](optional)",
+	Short: "List Charts assets versions state for release process",
+	RunE: func(cmd *cobra.Command, args []string) error {
+
+		var branch, chart string = "", ""
+
+		if len(args) < 1 {
+			return errors.New("expected at least one argument: [branch]")
+		}
+		branch = args[0]
+
+		if len(args) > 1 {
+			chart = args[1]
+		}
+
```

Comment:
Should we have any validations on input here? Ie validate a valid branch or chart has been provided?

Reply:
So, you got me thinking here, if a wrong chart is provided, this is the response:
```
Error: failed to execute charts-build-scripts: exit status 1, output: time="2024-07-26T18:33:23-03:00" level=info msg="Initializing dependencies for lifecycle-status"
time=2024-07-26T18:33:23-03:00 level=fatal msg=encountered error while initializing dependencies: no assets found in the repository

error: failed to execute charts-build-scripts: exit status 1, output: time="2024-07-26T18:33:23-03:00" level=info msg="Initializing dependencies for lifecycle-status"
time=2024-07-26T18:33:23-03:00 level=fatal msg=encountered error while initializing dependencies: no assets found in the repository
```
I agree this is no good, so

I have two options here: 
- either I change the response at `charts-build-scripts` when a wrong chart is provided or,
- I implement a scan of ecm-distro-tools release list charts at the `charts-repo`

Reply:
This is fine for a first pass but let's get an issue created for this and evaluate the best course of action later.